### ISSUE TITLE
[bug] SystemPackageTool installed() method is missing

### ISSUE THREAD

**Issue by Hopobcn** [conan-io/conan#6196]
According to [conan-io/docs](https://github.com/conan-io/docs/blame/18d6adbf56a55a7d9185a12aa707b5fe161b35e9/reference/conanfile/methods.rst#L642), [docs.conan.io](https://docs.conan.io/en/latest/reference/conanfile/methods.html#systempackagetool) SystemPackageTool should have a public `installed(package_name)` method. Instead, only a protected `_installed(packages)` is provided (see [here](https://github.com/conan-io/conan/blob/develop/conans/client/tools/system_pm.py#L146)).

### Environment Details (include every applicable attribute)
  * Operating System+version: Ubuntu 18.04.3 LTS
  * Compiler+version: no applicable
  * Conan version: 1.20.5
  * Python version: 3.6.9

### Steps to reproduce (Include if Applicable)
Try to use `SystemPackageTool.installed(package_name)`

### Logs (Executed commands with output) (Include/Attach if Applicable)
` ERROR: while executing system_requirements(): 'SystemPackageTool' object has no attribute 'installed'`

and if `_installed` is used:
```
Linter warnings
    WARN: Linter. Line 25: Access to a protected member _installed of a client class
```

**Comment by memsharded** [comment]
Hi @Hopobcn 

Yes, this is surprisingly true. Apparently it was introduced in the docs, but not in the code. I have submitted a PR https://github.com/conan-io/conan/pull/6198, lets see if it gets in 1.21 (it was already branched, maybe we need to wait until next 1.22). Thanks for reporting this issue!

**Comment by memsharded** [comment]
Closed by #6198 will be in next 1.21
